gatling-ecs-client/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM node:16.3.0-slim
 
-LABEL version=0.1.1
+LABEL version=0.1.2
```

Comment:
https://github.com/chatwork/dockerfiles/blob/master/gatling-ecs-client/Makefile#L16-L19

1. Build this image
2. Tagging to image from built image's version label